cmd/kots/cli/velero.go

```diff
@@ -30,20 +35,62 @@ func EnsurePermissionsCmd() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			v := viper.GetViper()
 
-			namespace := v.GetString("namespace")
-			if err := validateNamespace(namespace); err != nil {
+			veleroNamespace := v.GetString("velero-namespace")
+			if err := validateVeleroNamespace(veleroNamespace); err != nil {
+				return err
+			}
+
+			kotsadmNamespace := v.GetString("kotsadm-namespace")
+			if err := validateKotsadmNamespace(kotsadmNamespace); err != nil {
 				return err
 			}
 
-			if err := snapshot.EnsureVeleroPermissions(namespace); err != nil {
+			if err := snapshot.EnsureVeleroPermissions(veleroNamespace, kotsadmNamespace); err != nil {
+				return err
+			}
+
+			if err := snapshot.EnsureVeleroNamespaceConfigMap(veleroNamespace, kotsadmNamespace); err != nil {
 				return err
 			}
 
 			return nil
 		},
 	}
 
-	cmd.Flags().StringP("namespace", "n", "", "namespace in which kots/kotsadm is installed")
+	cmd.Flags().String("kotsadm-namespace", "", "namespace in which kots/kotsadm is installed")
```

Comment:
why not just `namespace`?

Reply:
wanted to distinguish between it and `velero-namespace`. no hard opinion here, i can change it back.